easybuild/easyconfigs/b/BCFtools/BCFtools-1.9-intel-2018b.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.9'
+
+homepage = 'http://www.htslib.org/'
+description = """Samtools is a suite of programs for interacting with high-throughput sequencing data.
+ BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence
+ variants"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['6f36d0e6f16ec4acf88649fb1565d443acf0ba40f25a9afd87f14d14d13070c8']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('HTSlib', '1.9'),
+    ('GSL', '2.5'),
```

Comment:
@boegel why have you removed `bzip2` and `XZ` as dependencies?

Reply:
They were also missing in the existing `BCFtools-1.9-foss-2018b.eb` (added in #6891), I don't see why, so re-added them in both easyconfigs.